codegenerator/cli/src/config_parsing/system_config.rs

```diff
@@ -1118,7 +1118,7 @@ impl EvmAbi {
 #[derive(Debug, Clone, PartialEq)]
 pub enum Abi {
     Evm(EvmAbi),
-    Fuel(FuelAbi),
+    Fuel(Box<FuelAbi>),
```

Comment:
I never used `Box`. What is it needed for?

Reply:
It's a clippy warning suggestion.

Box is like a c pointer. So it's an address on the heap. The reason to use it here is that the FuelAbi type is very big in memory. So the enum has to allocate all that memory even in the evm case. Clippy suggests using a pointer here instead to be more efficient